### PR TITLE
Adding Carissa's Coordinates

### PR DESCRIPTION
**Pull request by CarissaThomas**
@githubteacher This is a test.

### CODE REVIEW

**Comment by humanmirjalali** [comment]
@CarissaThomas Great job! 👍

**Comment by rvbarreto** [comment]
The check is failing because there are spaces in the beginning of each line. When you remove then, it will pass the test!

**Comment by kristinka545** [comment]
Good job!

File: _pins/CarissaThomas.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle:YOUR-USERNAME
```
Comment (crichID) [inline _pins/CarissaThomas.json]: @CarissaThomas you will want to replace `YOUR-USERNAME` with your GitHub username. You will also need to add a space after the `:` on each line!

File: _pins/CarissaThomas.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+latitude:YOUR_LATITUDE
```
Comment (crichID) [inline _pins/CarissaThomas.json]: @CarissaThomas similar to above, replace `YOUR_LATITUDE` with your actual coordinates and put a space after the `:`.

File: _pins/CarissaThomas.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+longitude:YOUR_LONGITUDE
```
Comment (crichID) [inline _pins/CarissaThomas.json]: @CarissaThomas same as above :wink:

File: _pins/CarissaThomas.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (crichID) [inline _pins/CarissaThomas.json]: @CarissaThomas you will need to add a line at the end just like the first line. It should contain 3 dashes `---`.

**Comment by crichID** [comment]
@CarissaThomas it looks like you had a good start, but there isn't enough information in your file for me to help you! I am going to go ahead and close out this pull request. If you would like to finish the activity, please feel free to re-open it or just start a new one!

**Pull request closed without merging**